cmd/podman/images/build.go

```diff
@@ -205,6 +206,24 @@ func build(cmd *cobra.Command, args []string) error {
 		return errors.New("'--output' option is not supported in remote mode")
 	}
 
+	if buildOpts.Network == "none" {
+		if cmd.Flag("dns").Changed {
+			return errors.New("the --dns option cannot be used with --network=none")
+		}
+		if cmd.Flag("dns-option").Changed {
+			return errors.New("the --dns-option option cannot be used with --network=none")
+		}
+		if cmd.Flag("dns-search").Changed {
+			return errors.New("the --dns-search option cannot be used with --network=none")
+		}
+	}
+
+	if cmd.Flag("network").Changed {
+		if buildOpts.Network != "host" && buildOpts.Isolation == buildahDefine.IsolationChroot.String() {
+			return fmt.Errorf("cannot set --network %s --isolation %s", buildOpts.Network, buildOpts.Isolation)
```

Comment:
Inconsistent with buildah:
```
# #| expected: =~ 'cannot set --network other than host with --isolation chroot'
# #|   actual:    'Error: cannot set --network none --isolation chroot'
```

Reply:
FYI options are change the message, skip the test (see treadmill PR), or use the `errmsg` directive in `apply-whatever` to change the expected error message. I think changing the podman error message is the easiest, but I don't have all the facts.